src/components/Table/Table.jsx

```diff
@@ -8,12 +8,14 @@ import {
   TableRowColumn as MaterialTableRowColumn,
```

Comment:
If possible, see if you can get this to work with React Router.

See https://github.com/odota/web/blob/master/src/components/Form/FormField.jsx for an example of a component that updates the URL via React Router.

You'll need the `history` object passed to this component, so you may need to add that as a prop where `Table` is used and you need pages

Once you do that you can just call history.push() to set the URL and I think you can also read out params from the URL from that object without writing your own url parsing code.

Reply:
if you go in matches/match.id/log and if you enter in the `FormField name='type'` or `name='players'` anything, it won't save the request in the URL

Reply:
yeah, I don't think the log UI is connected to the URL right now.  That's a separate issue, though